### ISSUE TITLE
S2758 Operation returns same value when true or false (False Positive)

### ISSUE THREAD

**Issue by njannink** [SonarSource/sonarlint-vs#412]
I get a false positive on the following code:

```
return MyObject.IsSelected ? $"{Value:0.00}" : $"{Value:c}";
```

**Comment by tvsonar** [comment]
@njannink Thanks for reporting this issue. I confirm that I could reproduce it. The problem comes from Roslyn itself, it is reporting the two interpolated strings to be the same. I'll investigate it a bit more and then report this issue to the Roslyn team.

**Comment by tvsonar** [comment]
related to https://github.com/dotnet/roslyn/issues/5271

**Comment by njannink** [comment]
Thank you

**Comment by tvsonar** [comment]
@njannink This issue seems to be fixed in Roslyn 1.1. I can't reproduce it anymore on Visual Studio 2015 Update 1.
